src/frontend/apps/impress/src/features/docs/docs-grid/components/DocsGridLoader.tsx

```diff
@@ -27,15 +26,11 @@ export const DocsGridLoader = ({ isLoading }: DocsGridLoaderProps) => {
         data-testid="grid-loader"
         $align="center"
         $justify="center"
-        $height="calc(100vh - 50px)"
+        $height="100%"
         $width="100%"
-        $maxWidth="960px"
-        $background="rgba(255, 255, 255, 0.3)"
+        $background="rgba(255, 255, 255, 0.5)"
         $zIndex={998}
-        $position="fixed"
-        $css={css`
-          top: ${HEADER_HEIGHT}px;
-        `}
+        $position="absolute"
```

Comment:
Changes related to css would require a separate commit for different explanations?

Reply:
You'r right was a mistake. You can read front so :D